.github/workflows/grader-notebook.yaml

```diff
@@ -0,0 +1,44 @@
+name: Grader Notebook
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Cache pip
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip
+        restore-keys: |
+          ${{ runner.os }}-pip-
+          ${{ runner.os }}-
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        cd grader-notebook
+        pip install nbgrader==0.6.1 psycopg2 black pytest pytest-coverage
```

Comment:
Why is a specific version of `nbgrader` required here?

Reply:
It is so far the only version I tested. As I did not have tests so far (but gradually building ones), testing another version would have been a lot of manual work.
Maybe I can remove this constraint, but so far it did not hurt

Reply:
Sure, but maybe it would be better to pin it to `>=0.6,<0.7` so that we can benefit from bug fixes.

Reply:
Just want to mention  that you are referring to the github action yaml - this is used for executing the tests - the more important one is probably the dockerfile. WIll change both.